services/app-web/src/pages/CMSDashboard/CMSDashboard.tsx

```diff
@@ -97,10 +62,16 @@ export const CMSDashboard = (): React.ReactElement => {
         }
     }
 
-    if (loginStatus === 'LOADING' || !loggedInUser || loading || !data) {
+    if (
+        loginStatus === 'LOADING' ||
+        !loggedInUser ||
+        loading ||
+        !data ||
+        !loggedInUser.__typename
```

Comment:
Type error for passing user `__typename` into `HealthPlanPackageTable` because `__typename` could could be `undefined`. Now that we are passing the entire user in, this isn't needed anymore.